src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> it's off topic for this discussion so I'll either forget about it or open a PR eventually.

No-no, lets not forget! If we don't do it, somebody else might - https://github.com/jquery/jquery/issues/2405